src/drivers/mct/cime_config/config_component.xml

```diff
@@ -1910,6 +1911,7 @@
 
   <entry id="NTASKS">
     <type>integer</type>
+    <default_value>1</default_value>
```

Comment:
default_value is a deprecated field.   The default, if defined, is the value below without attribute.